tests/tests/smtpd_mock.py

```diff
@@ -21,5 +21,9 @@ def __init__(self, *args, **kwargs):
         self.received = False
         smtpd.SMTPServer.__init__(self, *args, **kwargs)
 
-    def process_message(self, *args, **kwargs):
+    def process_message(self, peer, mailfrom, rcpttos, data):
+        print(peer)
```

Comment:
I was thinking about asserting sth about the message, like title and len>0. but I'm ok with what we have here, so it's an 'approve'. we have all of this covered pretty well.